packages/rollup-tests/src/ignored-passed-snapshot-different-tests.js

```diff
@@ -112,15 +112,15 @@ module.exports = [
     "rollup@form@exported-class-declaration-conflict: handles exporting class declarations with name conflicts in SystemJS@generates es",
     "rollup@form@external-empty-import-no-global: does not expect a global to be provided for empty imports (#1217)@generates es",
     "rollup@form@external-imports: prefixes global names with `global.` when creating UMD bundle (#57)@generates es",
-    "rollup@form@super-classes@super-class-prototype-assignment: correctly resolves the prototype of the super class when assigning properites",
+    "rollup@form@super-classes@super-class-prototype-assignment: correctly resolves the prototype of the super class when assigning properties",
```

Comment:
These typos came from rollup repository, which we clone from `rollup`, We don't plan to touch it.